public/stylesheet.css

```diff
@@ -634,21 +637,23 @@ body:not(.dark):not(.light) #color-scheme-system:not(:disabled) {
 }
 
 .grid-item__button {
+  fill: var(--color-button-active);
   border-top: 1px solid var(--color-grid-item-divider);
   background-color: var(--color-background);
-  color: var(--color-button-active);
   flex-grow: 1;
   font-family: var(--font-family-stylized);
   font-size: 0.75rem;
   font-weight: 600;
-  outline: none;
```

Comment:
@jorgeamadosoria

Reply:
@jorgeamadosoria, it appears this change is still here. If you have a reason to remove it, please share it. If not, please put it back.

Reply:
> ![image](https://user-images.githubusercontent.com/15719479/139752088-01259f6d-be54-41b8-873a-6d960a47b462.png)
> 
> I remove it because it doesn't seem to do anything.

Your UI says your comment is **Pending**, so I can't see it :wink: 

That said, it does affect rendering. All (major) browsers by default add `outline`s to tabable elements as a visual indicator of what element on the page is currently in focus. As we have our own styling for that we remove the outline (a decision open to discussion, but ideally we don't change that until a discussion as taken place AND consistently across the website in one go).

Reply:
@jorgeamadosoria, did you see my latest comment?

Reply:
I hadn't.
Outline didn't appear, after fixing the menu in the other PR I saw what that is. I can restore outline, no problem